app/views/corporate/index.html.erb

```diff
@@ -15,7 +15,7 @@
           </a>
         </div>
         <div class="l-promo-corporate__item">
-          <a class="promo__link" href="/en/corporate_categories/media-centre">
+          <a class="promo__link" href="/en/corporate/media-centre">
```

Comment:
I understand the static url change, but is it possible to use rails routing? Does `corporate_article_path('media-centre')` works?